src/components/surveys/survey-design/question-edit/rhs-subcontrols/Select.tsx

```diff
@@ -148,12 +148,24 @@ const Select: React.FunctionComponent<{
 
     const isSwitchedToSingleSelect = !step.singleChoice && value === 'SINGLE_SELECT'
 
+    const updatedStep = {...step}
+    let choices = isSwitchedToSingleSelect
+    ? step.choices.filter(c => !SurveyUtils.isSpecialSelectChoice(c))
+    : step.choices
+    if (!isSwitchedToSingleSelect) {
+      //if changing to a multiple select -- remove 'other'
```

Comment:
```suggestion
      // if changing to a multiple select -- remove 'other'
```

Reply:
Yeah, this is a weird one. All of Alina's comments don't have a space after the `//` and I thought maybe that was just the standard syntax for code comments?

Reply:
We generally use a space after the `//`, so as far as I know, the missing space isn't a standard syntax.

Reply:
thanks